receiver/cloudflarereceiver/config.go

```diff
@@ -0,0 +1,69 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cloudflarereceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/cloudflarereceiver"
+
+import (
+	"errors"
+	"fmt"
+	"net"
+
+	"go.opentelemetry.io/collector/config/configtls"
+	"go.uber.org/multierr"
+)
+
+// Config holds all the parameters to start an HTTP server that can be sent logs from CloudFlare
+type Config struct {
+	Secret            string                      `mapstructure:"secret"`
+	Endpoint          string                      `mapstructure:"endpoint"`
+	TLS               *configtls.TLSServerSetting `mapstructure:"tls"`
+	FieldAttributeMap map[string]string           `mapstructure:"field_attribute_map"`
```

Comment:
WDYT? Or maybe even just `attributes`?
```suggestion
	AttributesMap map[string]string           `mapstructure:"attributes_map"`
```